packages/grid/_modules_/grid/models/gridExport.ts

```diff
@@ -79,6 +85,11 @@ export interface GridPrintExportOptions {
    * Provide Print specific styles to the print window.
    */
   pageStyle?: string | Function;
+  /**
+   * If `true`, the Print export option will be removed from the GridToolbarExport menu.
+   * @default false
+   */
+  disable?: boolean;
```

Comment:
Maybe it's too premature, but this attribute could be extract into an interface so every export option must inherit it and support `disable`.

Reply:
Done in this commit: https://github.com/mui-org/material-ui-x/pull/3270/commits/9cbc962c80c04174791a5b5489f8d0b4f5964e7b

I took the common options: 
- fields
- allColumns
- disable

The only problem is for documentation comment, in which I have to replace "Print export" and "CSV export" by "this export"